circle/test.env

```diff
@@ -0,0 +1,8 @@
+SECRET_KEY_BASE=$SECRET_KEY_BASE
+OMNIAUTH_CLIENT_SECRET=$OMNIAUTH_CLIENT_SECRET
+OMNIAUTH_CLIENT_ID=$OMNIAUTH_CLIENT_ID
+DEVISE_SECRET_KEY=$DEVISE_SECRET_KEY
+AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+AWS_REGION=$AWS_REGION
+SQS_QUEUE_URL=$SQS_QUEUE_URL
```

Comment:
This gets the environmental variables for tests that get run on CircleCI
through Compose. I'll alter these for tests.

On Tue, Aug 25, 2015 at 9:53 AM, Omar Sahyoun <notifications@github.com>
wrote:

> In circle/test.env
> <https://github.com/SumOfUs/Champaign/pull/85#discussion_r37889537>:
>
> > @@ -0,0 +1,8 @@
> > +SECRET_KEY_BASE=$SECRET_KEY_BASE
> > +OMNIAUTH_CLIENT_SECRET=$OMNIAUTH_CLIENT_SECRET
> > +OMNIAUTH_CLIENT_ID=$OMNIAUTH_CLIENT_ID
> > +DEVISE_SECRET_KEY=$DEVISE_SECRET_KEY
> > +AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
> > +AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
> > +AWS_REGION=$AWS_REGION
> > +SQS_QUEUE_URL=$SQS_QUEUE_URL
>
> What does test.env do? Is it setting environment variables just for
> testing, using different variables for deployment?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SumOfUs/Champaign/pull/85/files#r37889537>.
>